gravitee-apim-distribution/pom.xml

```diff
@@ -1505,6 +1505,32 @@
                         </exclusion>
                     </exclusions>
                 </dependency>
+                <dependency>
+                    <groupId>com.graviteesource.connector</groupId>
+                    <artifactId>gravitee-entrypoint-native-kafka</artifactId>
+                    <version>${gravitee-connectors-native-kafka.version}</version>
+                    <type>zip</type>
+                    <scope>runtime</scope>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>*</groupId>
+                            <artifactId>*</artifactId>
+                        </exclusion>
+                    </exclusions>
+                </dependency>
+                <dependency>
+                    <groupId>com.graviteesource.connector</groupId>
+                    <artifactId>gravitee-endpoint-native-kafka</artifactId>
+                    <version>${gravitee-connectors-native-kafka.version}</version>
+                    <type>zip</type>
+                    <scope>runtime</scope>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>*</groupId>
+                            <artifactId>*</artifactId>
+                        </exclusion>
+                    </exclusions>
+                </dependency>
```

Comment:
Kafka reactor and native connectors will be part of the default bundle. You can move this to `bundle-default` profile

Reply:
Done 👍